src/main/scala/akka/contrib/http/Directives.scala

```diff
@@ -31,7 +32,7 @@ object Directives {
    */
```

Comment:
@ktoso - I need to release a version of akka-contrib-extra so I can proceed with upgrading ConductR to Akka Streams and HTTP 2.

Hopefully it's ok with you if we move on with the `accept` directive as a separate issue (I can raise them within akka-contrib-extra as well if you wish).